### DISCUSSION TITLE
Structure and workflow of this Github Repo

### DISCUSSION THREAD

**Discussion by Phyloiota** [iota-community/Community-Governance#17] [General]
Please suggest and discuss how we all should work together here...

**Comment by sascha1337** [comment]
i will fork my own 2018 created IOTA x IFPS x WebTorrent DAO platform to run on chrysalis instead of IRI legacy,  the base layer2 structure for a platform, similar as http://commonwealth.im/ - could any1 investigate if their platform would support hosting some non substrate (polkadot) based project,  too? 
I think i did also had contact previously with the guys that wrote that platform. And i know, the platform itself is fully centralized nature ( commonwealth ), but i love the simplicity and features on there.But the important features - the treasury ( not only one actually) funds, voting and actual proposals are all completely ONCHAIN and pretty advanced. Worth a look.

**Comment by Phyloiota** [comment]
Workflow and structure of changes to this Repo are now defined in the 

CONTRIBUTE file 

Have a look here: 

https://github.com/iota-community/Community-Governance/blob/main/.github/CONTRIBUTING.md
